### PR TITLE
Bugfixes 10 19

### LINKED ISSUE #68: Use auth0 refreshToken to keep a user logged in during a meeting
## Issue - Enhancement

It would be a stinky user experience of a user's JWT authentication token expired during a meeting. Instead we could:

- [ ] Request a refreshToken on behalf of the user when they log in
- [ ] If the JWT expires, and the user is attending a meeting, request a new JWT for them instead of asking them to reauthenticate

- **Estimated effort:** 5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### LINKED ISSUE #393: Do not require enter key to advance check-in
## Issue - Enhancement

When performing the check-in phase of the meeting, I want to simply press a key (**P**resent or **N**ot present) to advance the card, so that I can save keystrokes.

Needing to press enter isn't quite obvious enough...

- **Estimated effort:** 1 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by mattkrick**
- fix #393
- fix #68

### CODE REVIEW

File: src/server/Html.js [hunk 1]
```diff
@@ -42,9 +41,7 @@ export default function Html({store, entries, StyleSheetServer, renderProps}) {
         {/* segment.io analytics */}
         <script type="text/javascript" dangerouslySetInnerHTML={{__html: segmentSnippet}}/>
         {/* sentry.io error reporting */}
-        {process.env.SENTRY_DSN_PUBLIC &&
```
Comment (mattkrick) [inline src/server/Html.js]: if this is undefined the client still assumes Raven exists, so it'll error

File: src/universal/modules/meeting/components/CheckInCards/CheckInCards.js [hunk 2]
```diff
@@ -6,34 +6,38 @@ import CheckInCard from 'universal/modules/meeting/components/CheckInCard/CheckI
...
-const makeCheckinPressFactory = (teamMemberId) => (isCheckedIn) => () => {
+const makeCheckinPressFactory = (teamMemberId, gotoNext) => (isCheckedIn) => () => {
   const options = {
     variables: {
       isCheckedIn,
       teamMemberId
     }
   };
   cashay.mutate('checkIn', options);
+  gotoNext();
```
Comment (mattkrick) [inline src/universal/modules/meeting/components/CheckInCards/CheckInCards.js]: yeah, it's 2 calls to the server, but websockets are great for short little messages.

File: src/server/Html.js (hunk 1 above)
Comment (jordanh) [inline src/server/Html.js]: ah, strong point. Probably best just to always pull it in (as you've done).

File: src/server/utils/serverConstants.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import ms from 'ms';
```
Comment (jordanh) [inline src/server/utils/serverConstants.js]: Interesting! There are probably some other thing from .env.example we could pull in here such as `INVITE_TOKEN_KEY_HASH_ROUNDS`. Next time!

**Pull request merged**